java/client/src/main/java/glide/api/models/commands/PopDirection.java

```diff
@@ -0,0 +1,15 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.ListBaseCommands;
+
+/**
+ * Enumeration representing element pop direction and count for the {@link ListBaseCommands#lmpop}
```

Comment:
```suggestion
 * Enumeration representing element pop direction to start counting for the {@link ListBaseCommands#lmpop}
```

Reply:
actually i think "and count" should just be removed since i had the count api keyword here in the very first draft hence the description and forgot to remove that